app/src/main/java/com/rentminder/ui/theme/Color.kt

```diff
@@ -3,9 +3,7 @@ package com.rentminder.ui.theme
 import androidx.compose.ui.graphics.Color
 
 val Purple200 = Color(0xFFBB86FC)
-val Purple500 = Color(0xFF6200EE)
```

Comment:
Removed colors that weren't used for cleaner code.

https://docs.sonarqube.org/latest/user-guide/clean-as-you-code/